ng2-components/config/webpack.common.js

```diff
@@ -50,18 +70,22 @@ module.exports = {
                 test: /\.ts$/,
                 loader: 'tslint-loader',
                 options: {
-                    emitErrors: true,
+                    emitErrors: false,
                     configFile: path.resolve(__dirname, './assets/tslint.json')
                 },
                 exclude: [/node_modules/, /bundles/, /dist/, /demo/]
             },
             {
                 test: /\.ts$/,
                 use: ['ts-loader?' + JSON.stringify({
+                    "logInfoToStdOut": true,
+                    "logLevel": "error",
+                    "configFileName": path.resolve(__dirname, '../tsconfig.json'),
                     "compilerOptions": {
                         "typeRoots": [
-                            "../node_modules/@types"
-                        ]}
+                            "./node_modules/@types"
```

Comment:
it should be taking `node_modules/@types` by default, I think this setting can be removed.